airbyte-integrations/connectors/source-airtable/source_airtable/spec.json

```diff
@@ -4,14 +4,119 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Airtable Source Spec",
     "type": "object",
-    "required": ["api_key"],
+    "additionalProperties": true,
```

Comment:
```suggestion
```
True is the default.

Reply:
It should be `True` to provide the backward capability.